apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -303,6 +303,9 @@
             <x-menuitem @click="showColumnFilterModal">
               <x-label>Show or hide columns</x-label>
             </x-menuitem>
+            <x-menuitem @click="openQueryTab">
+              <x-label>Open in query tab</x-label>
```

Comment:
Not sure what copy we should use here.

We can use one of two conventions:
- How we have 'apply' and 'copy to sql' when performing an action (like editing)
- How we have `SQL: xyz` when right clicking a table.

I'm included to use the second convention here because this isn't the secondary action on an `apply` button.

Options:
- :cow:  `Open in query tab`
- :cat:  `Open table filter as SQL`
- :dog: `Copy to SQL`
- :deer: `Copy view to SQL`
- :snail:  `SQL: Table query`
- :skunk:  `SQL: Table view`

@not-night-but @wmontgomery @azmy60 vote with the emoji as a response to this comment!

Reply:
My vote: :deer:

Reply:
My vote: 🦌

Reply:
I vote 🦌, it looks right by convention.

Reply:
🦌 is my vote. Would it make sense to change "view" to whatever it is whether it's "table" or "view"?